src/gnome-software.ui

```diff
@@ -312,6 +312,7 @@
             <child>
               <object class="GtkStack" id="stack_main">
                 <property name="visible">True</property>
+                <property name="homogeneous">False</property>
```

Comment:
@GeorgesStavracas , while this fixes the low-res prob partially for some views, it also breaks the non-composite use a bit by not having homogeneous sizes making the window sometimes increase its size, right?
I would rather get the real fix for the views that are broken than creating a new problem for partially fixing one.